code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -844,13 +852,42 @@
 		M.reagents.add_reagent("toxin", reac_volume)
 	..()
 
+/datum/reagent/diamond
+	name = "Diamond"
+	id = "diamond"
+	description = "A girl's best friend."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	taste_description = "your wallet dying inside"
+	produce_type = /obj/item/stack/sheet/mineral/diamond
+	attack_force = 15
+	pick_speed = 8
+	penetration_value = 25
+	sharp_result = TRUE
+
+/datum/reagent/adamantine
+	name = "Adamantine"
+	id = "adamantine"
+	description = "STRIKE THE EARTH"
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	taste_description = "demons"
+	produce_type = /obj/item/stack/sheet/mineral/adamantine
+	attack_force = 20
+	pick_speed = 3
+	penetration_value = 40
+	sharp_result = TRUE
+
 /datum/reagent/uranium
 	name ="Uranium"
 	id = "uranium"
 	description = "A silvery-white metallic chemical element in the actinide series, weakly radioactive."
 	reagent_state = SOLID
 	color = "#B8B8C0" // rgb: 184, 184, 192
 	taste_description = "the inside of a reactor"
+	produce_type = /obj/item/stack/sheet/mineral/uranium
+	attack_force = 2
+	pick_speed = 15
```

Comment:
He's right, i though this was already done in a earlier commit, are you keeping track of what files you are uploading?